### PR TITLE
[RBrowser] tabs canvas selection

### PR DESCRIPTION
**Pull request by Falcort**
This PR do that when drawing or adding a canvas, selected tab will change to the correct one

### CODE REVIEW

File: gui/browserv7/src/RBrowser.cxx [hunk 1]
```diff
@@ -452,7 +452,9 @@ std::string ROOT::Experimental::RBrowser::ProcessDblClick(const std::string &ite
 
       canv->ForceUpdate(); // force update async - do not wait for confirmation
 
-      return "";
+      std::string res = "SLCTCANV:";
+      res.append(canv->GetName());
+      return res;
```
Comment (linev) [inline gui/browserv7/src/RBrowser.cxx]: While we using string_literals, one can simply do:
```
return "SLCTCANV:"s + canv->GetName();
```

File: gui/browserv7/src/RBrowser.cxx [hunk 2]
```diff
@@ -475,7 +477,9 @@ std::string ROOT::Experimental::RBrowser::ProcessDblClick(const std::string &ite
 
       rcanv->Update(true);
 
-      return "";
+      std::string res = "SLCTCANV:";
+      res.append(rcanv->GetTitle());
+      return res;
```
Comment (linev) [inline gui/browserv7/src/RBrowser.cxx]: Same here

File: ui5/browser/controller/Browser.controller.js [hunk 3]
```diff
@@ -461,6 +461,15 @@ sap.ui.define(['sap/ui/core/Component',
             var arr = JSON.parse(msg);
             this.createCanvas(arr[0], arr[1], arr[2]);
             break;
+         case "SLCTCANV": // Selected the back selected canvas
+           let oTabContainer = this.byId("myTabContainer");
+           let oTabContainerItems = oTabContainer.getItems();
+           for(let i=0; i<oTabContainerItems.length; i++) {
+             if (oTabContainerItems[i].getAdditionalText() === msg) {
+               oTabContainer.setSelectedItem(oTabContainerItems[i]);
+             }
```
Comment (linev) [inline ui5/browser/controller/Browser.controller.js]: add `break;` here

Comment (Falcort) [reply ui5/browser/controller/Browser.controller.js]: Solve in : fe09b4c

File: gui/browserv7/src/RBrowser.cxx (hunk 2 above)
Comment (Falcort) [reply gui/browserv7/src/RBrowser.cxx]: Solved in : fe09b4c

File: gui/browserv7/src/RBrowser.cxx (hunk 1 above)
Comment (Falcort) [reply gui/browserv7/src/RBrowser.cxx]: Solved in : fe09b4c

**Comment by linev** [comment]
Main code change is on client side, therefore no need to wait for jenkins.

**Pull request merged**